app/Member.php

```diff
@@ -206,6 +211,14 @@ public function getListofactionsAttribute()
 			return $item['date'];
 		});
 
+		if ($this->hasstraightflush) {
+			$data[] = [
+				'date' => Carbon::now(),
+				'name' => 'Straat',
+				'points' => 3
+			];
+		}
```

Comment:
Nette oplossing, maar op dit moment staat dit er al wel, maar in de view. Zie `members.show`:

```
@if ($member->hasstraightflush)
<tr>
	<td></td>
	<td>Straight flush</td>
	<td>3</td>
</tr>
@endif
```

Dus pas dit dan ook even aan :)

Reply:
In de tijd dat ik het puntensysteem implementeerde had ik nog niet geleerd om netjes te programmeren 😂

Reply:
Check ik haal hem uit de blade. Ik heb liever de data netjes op orde dan... jaaaa

Reply:
I made it better